test/test_land_cover.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+from onsset import SettlementProcessor
+
+from pandas import DataFrame,Series
+from pandas.testing import assert_frame_equal, assert_series_equal
+from pytest import fixture
+
+
+class TestSettlementProcessor:
+
+    @fixture
+    def setup_settlementprocessor(self) -> SettlementProcessor:
+
+        csv_path = os.path.join('test', 'test_data', 'dj-test.csv')
+        settlementprocessor = SettlementProcessor(csv_path)
+
+        return settlementprocessor
+
+    @fixture
+    def setup_dataframe(self) -> DataFrame:
+        df = DataFrame({
+                        'LandCover':list(range(17))
+                                               
+                         })
+        return df
+   
+    def test_classify_land_cover(self,setup_settlementprocessor,setup_dataframe):
+        sp = setup_settlementprocessor
+        df = setup_dataframe
```

Comment:
If you remove the `setup_dataframe` fixture as suggested above, this line would read:
```python
 df = DataFrame({'LandCover':list(range(17))})
```